src/cdk/a11y/focus-trap/focus-trap.ts

```diff
@@ -207,7 +209,10 @@ export class FocusTrap {
 
     if (redirectToElement) {
       // @breaking-change 8.0.0
-      if (redirectToElement.hasAttribute(`cdk-focus-initial`)) {
+      if (
+        (typeof ngDevMode === 'undefined' || ngDevMode) &&
```

Comment:
The warnings should be gated on dev mode, but I don't think this behavior here should be

Reply:
Unless I miss anything, this is just the warning being omitted in production mode. that seems reasonable to me.

Reply:
Nevermind, I misread the change